dmutils/apiclient/data.py

```diff
@@ -42,6 +48,21 @@ def acknowledge_audit_event(self, audit_event_id, user):
                 }
             })
 
+    def create_audit_event(self, audit_type, user, data, object_type=None, object_id=None):
```

Comment:
The object type/id is slightly odd in that these are DB model objects normally. I don't know we'd know from an API point of view? Wondering whether these can be derived from audit-type in some way? That is if I make a "register-interest" audit type, those are always associcated with a supplier, and what the API takes is the ID of whatever object the audit code will associate  the event with.

What do you think? Is it not scalable? 

I guess the value here is a string, and I'd know what object I want to associate the event with, so I have control?